src/patternfly/components/Hero/hero.scss

```diff
@@ -1,43 +1,59 @@
 @use '../../sass-utilities' as *;
 
 @include pf-root($hero) {
+  --#{$hero}--PaddingBlockStart: var(--pf-t--global--spacer--xl);
+  --#{$hero}--PaddingBlockEnd: var(--pf-t--global--spacer--xl);
+  --#{$hero}--PaddingInlineStart: var(--pf-t--global--spacer--3xl);
+  --#{$hero}--PaddingInlineEnd: 0;
   --#{$hero}--gradient--angle: 109deg;
   --#{$hero}--gradient--stop-1--light: transparent;
   --#{$hero}--gradient--stop-2--light: transparent;
   --#{$hero}--gradient--stop-3--light: transparent;
   --#{$hero}--gradient--stop-1--dark: transparent;
   --#{$hero}--gradient--stop-2--dark: transparent;
   --#{$hero}--gradient--stop-3--dark: transparent;
+  --#{$hero}--m-glass--BackgroundColor: var(--pf-t--global--background--color--glass--default);
+  --#{$hero}--m-glass--BackdropFilter: var(--pf-t--global--background--color--glass--filter);
```

Comment:
You shouldn't need `--m-glass` in these vars. They're regular vars, and they change at the global level when the glass theme is turned on.